config/routes.rb

```diff
@@ -241,6 +241,22 @@
 
   get '/exhibit/:ontology/:id' => 'concepts#exhibit'
 
+  # Admin
```

Comment:
this endpoint does already exist
https://github.com/ontoportal-lirmm/bioportal_web_ui/blob/7d05de5daf1d850ac2952d907d5f59fba27166f7/config/routes.rb#L96-L105

Reply:
Ah yes, I thought that someone did delete all of them by errror